Snowdog_Acm/templates/content/widget/sliders/list.phtml

```diff
@@ -0,0 +1,171 @@
+
+    foreach ($sliderBanners as $index => $banner) {
+        foreach ($pictureData as $pictureDataAttr) {
+            $bannersArray[$index][$pictureDataAttr] = $banner->getData($picturePrefix . $pictureDataAttr);
+        }
+
+        foreach ($pictureFiles as $pictureFile) {
+            $bannersArray[$index][$pictureFile] =
+                $banner->getData($picturePrefix . $pictureFile) ? $banner->getFile($picturePrefix . $pictureFile) : false;
+        }
+    }
+
+    // sorting banners collection by order
+    usort($bannersArray, function ($a, $b) {
+        return $a['order'] - $b['order'];
+    });
+    ?>
+
+    <?php
+    $sliderBlockBefore->setData([
+        'slider_html'       => 'before-slides',
+        'slider_class'      => $sliderClassName,
+        'wrapper_class'     => $sliderWrapperClass,
+        'display_title'     => $sliderDisplayTitle,
+        'slider_title'      => $sliderTitle,
+        'title_class'       => $sliderTitleClassName,
+        'content_before'    => $sliderContentBefore,
+        'infinite'          => $sliderInfinite,
+        'center_mode'       => $sliderCenterMode,
+        'dots'              => $sliderDots,
+        'arrows'            => $sliderArrows,
+        'autoplay'          => $sliderAutoplay,
+        'autoplay_speed'    => $sliderAutoplaySpeed,
+        'pause_on_focus'    => $sliderPauseOnFocus,
+        'pause_on_hover'    => $sliderPauseOnHover,
+        'slides_to_show'    => $sliderSlidesToShow,
+        'slides_to_scroll'  => $sliderSlidesToScroll,
+        'responsive_config' => $sliderResponsiveConfig
+    ]);
+    ?>
+    <?= $sliderBlockBefore->toHtml(); ?>
+
+    <?php if ($bannersArray): ?>
+        <?php foreach ($bannersArray as $banner): ?>
+            <?php
+            $bannerUrl = $block->escapeUrl($banner['url']);
+            $bannerUrlTarget = $block->escapeUrl(($banner['url_target'] == 'blank') ? '_blank' : '_self');
+
+            $sliderPictureBlock->setData([
+                'picture_class' => 'banner__image',
+                'picture_full_url' => true,
+                'img_default' => $banner['default'],
+                'img_480' => $banner['max_480'],
+                'img_768' => $banner['max_768'],
+                'img_1024' => $banner['max_1024'],
+                'img_1328' => $banner['max_1328'],
+                'img_full' => $banner['full'],
+                'img_4802x' => $banner['max_480_2x'],
+                'img_7682x' => $banner['max_768_2x'],
+                'img_10242x' => $banner['max_1024_2x'],
+                'img_13282x' => $banner['max_1328_2x'],
+                'img_full_2x' => $banner['full_2x'],
+                'picture_alt' => $banner['alt']
+            ]);
+            ?>
+            <div class="<?= $block->escapeHtmlAttr($sliderItemClass) ?>">
+                <<?= $bannerUrl ? 'a href="' . $bannerUrl . '" target="' . $bannerUrlTarget . '"' : 'div'; ?>
```

Comment:
For href value we should add `/* @noEscape */` (unless we will remove escapeHtml from variable declaration and use escapeHtml method directly here).